reference/security.rst

```diff
@@ -131,53 +112,28 @@ The following Go packages are used to implement this functionality:
 Authorisation Code Flow
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-When performing login via a browser, users go through the authorisation code flow.
-This is best described by various diagrams such as 
-`this <https://auth0.com/docs/get-started/authentication-and-authorization-flow/authorization-code-flow>`__.
+In a browser-based login, users follow the `authorisation code flow <https://auth0.com/docs/get-started/authentication-and-authorization-flow/authorization-code-flow>`__.
+Upon successful login, the access and refresh tokens are stored on the backend, 
+and a session cookie is issued to the user's browser.
 
-In this flow the user's browser is redirected to the identity provider where they are
-asked to login before being redirected back to the original application.
-
-Because JAAS is a server side app, the access token and refresh token are stored
-by the back-end application and a session cookie is issued to the browser as described in 
-`Browser Cookies`_.
-
-To protect against `CSRF attacks <https://auth0.com/docs/secure/attack-protection/state-parameters>`__
-the back-end application issues a random nonce used in the ``state`` parameter of the OAuth
-flow. This prevents a malicious attacker from forging a request to login as another user.
+To protect against `CSRF attacks <https://auth0.com/docs/secure/attack-protection/state-parameters>`__,
+the backend issues a random nonce in the ``state`` parameter of the OAuth flow.
 
 Device Code Flow
 ^^^^^^^^^^^^^^^^
 
-When performing login via the device flow, the Juju CLI will:
-
-1. Request from JAAS a URL and random code which the user can use to login.
-2. The CLI will then wait for a response from JAAS.
-3. JAAS polls the identity server, which in turn is waiting for the user to complete their login.
-
-A diagram depicting this login flow can be found 
-`here <https://auth0.com/docs/get-started/authentication-and-authorization-flow/authorization-code-flow>`__.
+For CLI logins, the `device code flow <https://auth0.com/docs/get-started/authentication-and-authorization-flow/device-authorization-flow>`__
+is used, where the Juju CLI prompts the user to log in via a browser. The backend stores access 
```

Comment:
I think we should also mention the random code (like as you did before).

Reply:
Added that bit back